demos/adjoint/inverse.py

```diff
@@ -76,7 +77,7 @@ def inverse(alpha_u, alpha_d, alpha_s):
     Tic = Function(Q1, name="Initial Temperature")
     Taverage = Function(Q1, name="Average Temperature")
 
-    checkpoint_file = CheckpointFile("Checkpoint_State.h5", "r")
```

Comment:
I guess this means you want to delete `Checkpoint_State.h5` from git and out of the requirements for the testing pipeline?

Reply:
We didn't have `Checkpoint_State.h5` on git, did we? I think the one `.h5` file we had was for the cylindrical case. This case starts from a UFL prescribed initial condition. The only thing I have added is that for the tutorials, the user can download it instead of running it from `forward.py`.

Reply:
Yeah, my mistake, sorry!